forms-flow-api/src/formsflow_api/resources/application.py

```diff
@@ -229,8 +229,14 @@ def get():  # pylint:disable=too-many-locals
         include_drafts = dict_data.get("include_drafts", False)
         only_drafts = dict_data.get("only_drafts", False)
         created_user_submissions = dict_data.get("created_user_submissions", False)
-
-        if auth.has_role([VIEW_TASKS, MANAGE_TASKS]) and not created_user_submissions:
+        form_name = ApplicationService.fetch_latest_form_name_by_parent_form_id(
+            common_filters["parent_form_id"]
+        )
+        # Check if the application_id is not a valid integer, return an empty response
+        application_id = dict_data.get("application_id")
+        if application_id and not application_id.isdigit():
```

Comment:
sometimes the applicationId will come proper integer then isdigit() will return error right?

Reply:
The request is processed using **ApplicationListRequestSchema()**, where the applicationId is defined as a string application_id = fields.Str(data_key="Id", required=False). As a result, the application_id will be treated as a string in this context.